code/modules/reagents/reagent_containers/bidon.dm

```diff
@@ -0,0 +1,172 @@
+
+//this is big movable beaker
+/obj/structure/reagent_dispensers/bidon
+	name = "B.I.D.O.N. canister"
+	desc = "Blue-Ink Dispenser Omnitech-Nanochem. A canister with acid-resistant linings intended for handling big volumes of chemicals."
```

Comment:
Ater pinged me about this being a bit questionable word salad like a month ago and the suggested replacement was:
Banded Industrial Drum for Organic Nonvolatiles
If Omnitech-Nanochem is a company you want namedropped it should probably be just in the desc and not the acronym.

Reply:
Why was this marked as resolved, no changes were made?

Reply:
Apologies, forgot to push a name change fix.